dask_sql/physical/rel/logical/join.py

```diff
@@ -178,6 +179,7 @@ def merge_single_partitions(lhs_partition, rhs_partition):
         # and to rename them like the rel specifies
         row_type = rel.getRowType()
         field_specifications = [str(f) for f in row_type.getFieldNames()]
+        print(field_specifications)
 
```

Comment:
In the `dask_sql/physical/rel/base.py` script, replace the `fix_column_to_row_type` method with
```
@staticmethod
    def fix_column_to_row_type(
        cc: ColumnContainer, row_type: "RelDataType", join_type: str = None,
    ) -> ColumnContainer:
        """
        Make sure that the given column container
        has the column names specified by the row type.
        We assume that the column order is already correct
        and will just "blindly" rename the columns.
        """
        field_names = [str(x) for x in row_type.getFieldNames()]
        if join_type == "leftanti":
            field_names = field_names[:len(cc.columns)]

        logger.debug(f"Renaming {cc.columns} to {field_names}")
        cc = cc.rename_handle_duplicates(
            from_columns=cc.columns, to_columns=field_names
        )

        # TODO: We can also check for the types here and do any conversions if needed
        return cc.limit_to(field_names)
```